release-notifier/internal/file/file.go

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+package file
```

Comment:
All functions in this package are related to PostgreSQL. 
Even though the information might be stored as a file, this is just a technical detail and therefore I think the package name is misleading. What would happen, if we decide to store the info in a different way? It would still be about PostgreSQL, but not anymore about files. So the better option for package names is always the business case and not the technical detail

Reply:
Renamed to 'cache'.